content/blog/2017/blocking-analytics-spam.md

```diff
@@ -21,17 +22,28 @@ summary: |
 date: 2017-03-23
 ---
 
+{% import 'embed.macros.njk' as embed %}
+
 Like many websites, we use [Google Analytics] to gather data about our
 users – what OS and browser they used, how they came to our site, etc.
 But a number of months ago we started seeing lots of this:
 
-<img src="{{ site.images }}blog/2017/ga-spam/ga-spam.jpg" class="img-border img-border" alt="google analytics spam" />
+{{ embed.img(
+  src='blog/2017/ga-spam/ga-spam.jpg',
+  alt='Google Analytics spam',
+    attrs={'class': 'img-border'}
```

Comment:
@mirisuzanne I notice we're using these `.img-border` and `.img-shadow` etc classes a number of places, but there don't seem to be corresponding styles to make them do anything?

Reply:
Those are from the old codebase, and we haven't taken the time to remove them. Much of the blog was ported using our migration tool, without much cleanup. We figured things like that can happen after launch?

Reply:
Makes sense -- Should they just be removed as I find them, or left because we want to replace them with similar new styles?

Reply:
We haven't been preserving them. Feel free to remove. We've also been moving images into either markdown or the `embed.img` macro when we get around to it - though again there's no urgency